content/en/docs/v3.6/learning/api.md

```diff
@@ -360,6 +360,8 @@ Watches make three guarantees about events:
 * Reliable - a sequence of events will never drop any subsequence of events; if there are events ordered in time as a < b < c, then if the watch receives events a and c, it is guaranteed to receive b.
 * Atomic - a list of events is guaranteed to encompass complete revisions; updates in the same revision over multiple keys will not be split over several lists of events.
 
+Note that these guarantees are not weaker than e.g. `Range`. For more information about the limitations, see [API guarantees][consistency].
```

Comment:
Don't understand the point about `Range` I would say that exactly `Watch` has weaker guarantees then `Range`.